tasks/main.yml

```diff
@@ -16,37 +16,60 @@
       - "--strip-components=1"
     creates: /var/guacamole/docker-compose.yml
 
+- name: Pull the Docker images used by the Guacamole Docker composition
+  block:
+    # Docker is only enabled in ansible-role-docker, but we need it to
+    # actually be running in order to pull Docker images.
+    - name: Start Docker
+      ansible.builtin.service:
+        name: docker
+        state: started
+
+    # Ideally we'd use the community.general.docker_compose Ansible
+    # module for pulling the images, but that module does not allow
+    # for pulling images without actually starting up the composition.
+    - name: >
```

Comment:
Why is this `>` needed here?

Reply:
The task name was long.  Using the [block style indicator](https://yaml-multiline.info/) to start the line a few spaces to the left on the next line buys me a little extra room.

Reply:
I thought it might be that, but when I eyeballed it, the task name didn't seem to be "too long", so I asked just in case I was missing something.  Thanks for the explanation.

Reply:
Maybe it's not "too long", but it was long enough that I felt like doing what I did.  Even our style guide must leave a _little_ room for artistic license!